README.md

```diff
@@ -53,13 +53,10 @@ bundle exec rake jasmine:ci
 ```
 
 ### Pact tests
-Collections Organisations API has pact tests with [gds-api-adapters](https://github.com/alphagov/gds-api-adapters/blob/19515f01395a2a2cdfa22e1c86f8cb1a4298c492/test/test_helpers/pact_helper.rb).
 
-Use `bundle exec rake pact:verify` to run the pact tests against the master pactfile stored on the pact broker.
+Collections Organisations API has a pact with [GDS API Adapters](https://github.com/alphagov/gds-api-adapters).
 
-If you have made a change to the code that consumes the organisations api, you might want to confirm that collections will still honour its pact. You can do so by running the pact tests against your local gds-api-adapters branch like so:
-- From gds-api-adapters run `bundle exec rake` to regenerate a new pactfile on you local machine.
-- From collections run `bundle exec rake PACT_URI="../gds-api-adapters/spec/pacts/gds_api_adapters-collections_organisation_api.json" pact:verify`
+See [the guidance](https://docs.publishing.service.gov.uk/manual/pact-broker.html) for how to run and modify the tests.
```

Comment:
Here's a new a11y suggestion from me: we now have two links with text as "the guidance", which is better than just "here" or "read more", but is still [quite vague when consumed in isolation](https://fae.disability.illinois.edu/rulesets/LINK_2/). This might be better:

```suggestion
Read [how to run and modify Pact tests](https://docs.publishing.service.gov.uk/manual/pact-broker.html).
```

Reply:
That's fair, particularly for the tech docs intro, which I'm expecting to be copied in future PRs for other frontend apps. I've amended both in the relevant commits.

How does it look now?